packages/core/src/components/Label/__stories__/label.mdx

```diff
@@ -144,6 +144,14 @@ In case of visual overload, use the secondary label in order to create hirarchy.
   <Story of={LabelStories.SecondaryLabel} />
 </Canvas>
 
+### Celebration
+
+To celebrate new feature, outline label can be highlighted by adding celebrate animation.
+
+<Canvas>
+  <Story of={LabelStories.Celebration} />
```

Comment:
I saw Yossi opened a PR with the deletion of the <story> tag, maybe change it to <Canvas of=